docs/security/index.mdx

```diff
@@ -0,0 +1,70 @@
+---
+title: Security
+displayed_sidebar: main
+---
+
+:::note
+This section is currently not exhaustive.
+:::
+
+This document describes the different security measures and deployment settings provided by Saleor.
+
+*Looking to report a security issue instead? See our [Security Policy].*
+
+## External Connectivity
+
+### HTTP Redirects and Timeouts
+
+Out of the box, Saleor disables HTTP redirects for outgoing connections, and sets a strict timeout value (typically \<20s).
+
+This is a design choice intended to prevent potential Denial of Service (DoS) attacks from being done by internal actors.
+
+Without such measure in place, a malicious staff user could, for example, create an infinite redirection loop in a malicious endpoint and exhaust server resources.
+
+Timeout settings cannot be overridden dynamically. However, they can be altered inside [`settings.py`](https://github.com/saleor/saleor/blob/1f2127e1e92722cecae7409c0d26ac7ab05d23d8/saleor/settings.py#L961-L968).
```

Comment:
**Question:** shoudn't we link to blob on `main` branch instead?

Reply:
If we were to link to a branch, then the link would unfortunately start linking to something unrelated over time 😕 (hence the full commit hash to ensure the contents don't mutate)